From cd4f7e18f3b15203afbf12a14b27f4cf5f779dd4 Mon Sep 17 00:00:00 2001
Message-Id: <cd4f7e18f3b15203afbf12a14b27f4cf5f779dd4.1367947969.git.minovotn@redhat.com>
In-Reply-To: <707b9b97153063374d2530e72c49b1499fc21af9.1367947969.git.minovotn@redhat.com>
References: <707b9b97153063374d2530e72c49b1499fc21af9.1367947969.git.minovotn@redhat.com>
From: Laszlo Ersek <lersek@redhat.com>
Date: Mon, 6 May 2013 19:27:25 +0200
Subject: [PATCH 060/114] qemu-ga: align versioning with QEMU_VERSION

RH-Author: Laszlo Ersek <lersek@redhat.com>
Message-id: <1367868499-27603-3-git-send-email-lersek@redhat.com>
Patchwork-id: 51101
O-Subject: [RHEL-6.5 qemu-kvm PATCH v2 02/56] qemu-ga: align versioning with QEMU_VERSION
Bugzilla: 952873
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Michael Roth <mdroth@linux.vnet.ibm.com>

Previously qemu-ga version was defined seperately. Since it is aligned
with QEMU releases, use QEMU_VERSION instead. This also implies the
version bump for 1.1[-rcN] release of qemu-ga.

Reviewed-by: Michal Privoznik <mprivozn@redhat.com>
Acked-by: Luiz Capitulino <lcapitulino@redhat.com>
Signed-off-by: Michael Roth <mdroth@linux.vnet.ibm.com>
(cherry picked from commit 8efacc43aef2c8fffceb0aaa52b8b7658a3e41ff)
Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 qga/guest-agent-core.h |    1 -
 qemu-ga.c              |    4 ++--
 qga/commands.c         |    2 +-
 3 files changed, 3 insertions(+), 4 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-ga.c              | 4 ++--
 qga/commands.c         | 2 +-
 qga/guest-agent-core.h | 1 -
 3 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/qemu-ga.c b/qemu-ga.c
index 2f7f598..e3c1929 100644
--- a/qemu-ga.c
+++ b/qemu-ga.c
@@ -169,7 +169,7 @@ static void usage(const char *cmd)
 "  -h, --help        display this help and exit\n"
 "\n"
 "Report bugs to <mdroth@linux.vnet.ibm.com>\n"
-    , cmd, QGA_VERSION, QGA_VIRTIO_PATH_DEFAULT, QGA_PIDFILE_DEFAULT,
+    , cmd, QEMU_VERSION, QGA_VIRTIO_PATH_DEFAULT, QGA_PIDFILE_DEFAULT,
     QGA_STATEDIR_DEFAULT);
 }
 
@@ -729,7 +729,7 @@ int main(int argc, char **argv)
             log_level = G_LOG_LEVEL_MASK;
             break;
         case 'V':
-            printf("QEMU Guest Agent %s\n", QGA_VERSION);
+            printf("QEMU Guest Agent %s\n", QEMU_VERSION);
             return 0;
         case 'd':
             daemonize = 1;
diff --git a/qga/commands.c b/qga/commands.c
index 5bcceaa..46b0b08 100644
--- a/qga/commands.c
+++ b/qga/commands.c
@@ -52,7 +52,7 @@ struct GuestAgentInfo *qmp_guest_info(Error **err)
     GuestAgentCommandInfoList *cmd_info_list;
     char **cmd_list_head, **cmd_list;
 
-    info->version = g_strdup(QGA_VERSION);
+    info->version = g_strdup(QEMU_VERSION);
 
     cmd_list_head = cmd_list = qmp_get_command_list();
     if (*cmd_list_head == NULL) {
diff --git a/qga/guest-agent-core.h b/qga/guest-agent-core.h
index 6dba104..49a7abe 100644
--- a/qga/guest-agent-core.h
+++ b/qga/guest-agent-core.h
@@ -13,7 +13,6 @@
 #include "qapi/qmp-core.h"
 #include "qemu-common.h"
 
-#define QGA_VERSION "1.0"
 #define QGA_READ_COUNT_DEFAULT 4096
 
 typedef struct GAState GAState;
-- 
1.7.11.7

